editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -6099,11 +6097,12 @@ void CanvasItemEditorViewport::drop_data(const Point2 &p_point, const Variant &p
 	Node *root_node = EditorNode::get_singleton()->get_edited_scene();
 	if (selected_nodes.size() > 0) {
 		Node *selected_node = selected_nodes.front()->get();
-		target_node = selected_node;
 		if (is_alt) {
 			target_node = root_node;
-		} else if (is_shift && selected_node != root_node) {
-			target_node = selected_node->get_parent();
+		} else if (is_shift) {
+			target_node = selected_node;
+		} else { // default behaviour
```

Comment:
```suggestion
		} else { // Default behavior.
```
For both, please use US English